build.go

```diff
@@ -66,16 +66,16 @@ func BuildFiles(payload Payload, dir string, options *BuildOptions) error {
 			return err
 		}
 
-		f, err := os.Create(path)
+		file, err := os.Create(path)
```

Comment:
I'm not sure why you are renaming existing variables. These changes do not seem to be relevant to the main change you want to make. I suggest you avoid doing this so that reviewers can focus on the change in functionality you wish to introduce and give you a higher quality review.

I too used to be bugged by the short, meaningless variables names that seem to be prevalent in Go, but [when they are short lived](https://research.swtch.com/names) it's generally OK.

Reply:
This is a very helpful feedback! I'll remove all the changes related to linter issues in this PR as long as they pass the linter check in the pipeline. Thanks!